modules/move-etcd-different-disk.adoc

```diff
@@ -10,20 +10,52 @@ You can move etcd from a shared disk to a separate disk to prevent or resolve pe
 
 The Machine Config Operator (MCO) is responsible for mounting a secondary disk for {product-title} {product-version} container storage.
 
+[NOTE]
+====
+This procedure supports NVME devices and devices where the name starts with `/dev/sd*` or `/dev/vd*`. For an NVME device, use `/dev/nvme*[0-9]\*n*` as the device name.
```

Comment:
I'm afraid I don't understand what this admonition means or is trying to say.

Are we saying that the NVME device name must start with `sd*` or `vd*`, or are we suggesting that the reader will have to replace `sd*`/`vd*` with `nvme[0-9]`?

Reply:
The end user needs to replace the disk name in the script as per their env.  We have given examples for sd* , vd* and for nvme they can use `/dev/nvme*[0-9]\*n*`